cmd/guaccollect/cmd/files.go

```diff
@@ -171,7 +171,6 @@ func initializeNATsandCollector(ctx context.Context, pubsubAddr string, blobAddr
 		if err := collector.Collect(ctx, emit, errHandler); err != nil {
 			logger.Errorf("Unhandled error in the collector: %s", err)
 		}
-		done <- true
```

Comment:
Don't remove this line, or the command will not exit when the collector errors.

Reply:
Whoops, I deleted that line by mistake!